app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt

```diff
@@ -112,36 +112,58 @@ class TopicRevisionFragmentTest {
   }
 
   private fun setUpTestApplicationComponent() {
-    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(
+      topicRevisionFragmentTest = this
```

Comment:
We do not require a named argument here as there's only a single argument here.

Reply:
+1

Reply:
Done